sonarqube/datadog_checks/sonarqube/check.py

```diff
@@ -28,12 +28,18 @@ def __init__(self, name, init_config, instances):
 
     def check(self, _):
         try:
+            self.collect_metadata()
             self.collect_metrics()
         except RequestException as e:
             self.service_check(self.SERVICE_CHECK_CONNECT, self.CRITICAL, tags=self._tags, message=str(e))
         else:
             self.service_check(self.SERVICE_CHECK_CONNECT, self.OK, tags=self._tags)
 
+    def collect_metadata(self):
+        response = self.http.get('{}/api/server/version'.format(self._web_endpoint))
+        response.raise_for_status()
+        self.collect_version(response)
```

Comment:
This change seems pretty unrelated

Reply:
'Sonar-Version' http header no more supported in new versions
The right way to get the version is using the endpoint `/api/server/version` as indicated in its documentation